dev/io.openliberty.checkpoint/resources/io/openliberty/checkpoint/resources/CheckpointMessages.nlsprops

```diff
@@ -37,3 +37,22 @@ RESTORE_FAILED_CWWKC0454E=CWWKC0454E: The server failed to restore from a checkp
 RESTORE_FAILED_CWWKC0454E.explanation=An attempt to restore the server from a checkpoint failed.  The server is not running.
 RESTORE_FAILED_CWWKC0454E.useraction=Inspect the checkpoint directory located in the server logs directory for additional log files to determine why the server could not be restored.
 
+UKNOWN_FAILURE_CWWKC0455E=CWWKC0455E: An unknown error occurred with the following message: {0}
```

Comment:
Please put more specific information in the message. For example, "A checkpoint request stopped with an error: {0}"

Did the request stop? Did it complete but generate an error?

Reply:
This error message is always emitted along with message CWWKC0453E (checkpoint failed) or CWWKC0454E (restore failed) depending on if unknown error type happened on checkpoint or on restore.  So you will see messages like this:

```
CWWKC0453E: The server checkpoint request failed with the following message: CWWKC0455E: An unknown error occurred with the following message: ...
```
or if on the restore side
```
RESTORE_FAILED_CWWKC0454E=CWWKC0454E: The server failed to restore from a checkpoint with the following message:  CWWKC0455E: An unknown error occurred with the following message: ...
```

The cause for the failure for restore or checkpoint is usually a known error type that we have a more specific message for.  These unknown ones would be used for unexpected ones, for example if we encounter some unexpected runtime error like a NullPointerException.

For other error types we would give more information.  For example, if we have a prepare hook throw an exception to indicate that the checkpoint should not proceed and instead fail we would have something like this:

```
[ERROR   ] CWWKC0453E: The server checkpoint request failed with the following message: CWWKC0457E: A error occurred preparing to take a checkpoint with the following message: TESTING - prepare hook fails.
```
Here that `TESTING - prepare hook fails.` is just some text from a test hook.  But hooks can also return their own messages for why they failed the checkpoint or restore.

Does that help give context for this?  I don't really want to add more information directly to this message because the proceeding part of the message will say what failed (checkpoint or restore) and the last part of the message will give the detail on what actually caused the failure.